cg/store/models.py

```diff
@@ -393,6 +393,7 @@ class Case(Model, PriorityMixin):
     internal_id = Column(types.String(32), unique=True, nullable=False)
     is_compressible = Column(types.Boolean, nullable=False, default=True)
     name = Column(types.String(128), nullable=False)
+    order_id = Column(ForeignKey("order.id", ondelete="CASCADE"))
```

Comment:
Will this delete all cases associated with an order? Maybe we want to skip it for now if we want to delete stuff in the order table, not sure! You know best

Reply:
We skip it for now!

Reply:
I will need to test so that we can remove cases still, even if they are in an order though. Or maybe we need to refine that condition? In a separate PR though.